src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -246,7 +247,7 @@ def from_dict(cls, r_dict):
 # Config options
 CLUSTER_MGR_ROLE = Role(
     'cluster-manager', """allows full permissions for the hosts, osd, mon, mgr,
-    and config-opt scopes.""", {
+    and config-opt scopes""", {
```

Comment:
I think this change isn't required, it's just a description.

Reply:
I removed the '.' to make it align with all the other descriptions that doesn't have '.' It isn't required but its a nit.